src/data.js

```diff
@@ -2449,6 +2449,26 @@ export default [
     phone: 'iphone',
     tags: ['JavaScript', 'Developer', 'TypeScript', 'React', 'Rust', 'Node'],
   },
+  {
+    name: 'Frank Bültge',
+    description: 'Alpinist, Cyclist, Optimist. I develop, support solutions, primarily with SAP at ZEISS Group and I make WordPress-based solutions at the Inpsyde',
+    url: 'https://bueltge.de',
```

Comment:
This needs to be a "uses" page 😄

Reply:
Fast feedback :)
For now the uses page is include.